src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -513,6 +516,43 @@ export class SearchInterface extends RootComponent implements IComponentBindings
   }
 
   /**
+   * Get the Query context for the current Search Interface.
+   * 
+   * If the search interface has performed at least one query, it will try to resolve the context from the last query sent to the Coveo Search API.
```

Comment:
Hm, are you sure you want to return the last query?

Say I am dynamically changing context values based on events in the page (like "Did the user add a rating to a product), this might not trigger a query but the context value is now different and the old one will still be used for query suggestions.